lib/graph/providers/NodePackageDependencies.js

```diff
@@ -99,15 +99,15 @@ class NodePackageDependencies {
 		}
 	}
 
-	async _getNode(modulePath, optional) {
+	async _getNode(modulePath, optional, nameAlias) {
```

Comment:
I think it would help us in the future to add a JSDoc comment here describing that this refers to the npm name alias concept

Reply:
Done

Reply:
Done